pkg/providers/cloudstack/cloudstack.go

```diff
@@ -501,7 +500,7 @@ func buildTemplateMapMD(clusterSpec *cluster.Spec, datacenterConfigSpec v1alpha1
 	values := map[string]interface{}{
 		"clusterName":                clusterSpec.ObjectMeta.Name,
 		"kubernetesVersion":          bundle.KubeDistro.Kubernetes.Tag,
-		"cloudstackNetwork":          datacenterConfigSpec.Network.Value,
+		"cloudstackNetwork":          datacenterConfigSpec.Zones[0].Network.Value,
```

Comment:
This will eventually change in a follow up PR to support multiple networks based on the zone? Or do we only support the first network defined?